docs/SSOT.md

```
@@ -0,0 +1,33 @@
+# SSOT — Tracera
+
+## State
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)